app/frontend/src/components/forestrysectoropscreening/Form.vue

```diff
@@ -104,13 +139,46 @@ export default {
   },
   data () {
     return {
+      landingValid: false,
+
+      // TODO: fetch this from API
+      operationTypes: [
+        { type:'BRUSHING', display: 'Brushing', enabled: true },
+        { type:'CONE_PICKING', display: 'Cone Picking', enabled: true },
+        { type:'ENGINEERING', display: 'Engineering', enabled: true },
+        { type:'LAYOUT', display: 'Layout/Timber Cruising', enabled: true },
+        { type:'LOGGING', display: 'Logging', enabled: true },
+        { type:'PRUNING', display: 'Pruning', enabled: true },
+        { type:'RESEARCH', display: 'Research', enabled: true },
+        { type:'SILVICULTURE', display: 'Silviculture', enabled: true },
+        { type:'SPACING', display: 'Spacing', enabled: true },
+        { type:'SURVEYING', display: 'Surveying', enabled: true },
+        { type:'VEGETATION_MANAGEMENT', display: 'Vegetation Management', enabled: true },
+        { type:'WILDFIRE_CAMPS', display: 'Wildfire Camps', enabled: true }
+      ]
     };
   },
```

Comment:
Minor style commenet: As this data doesn't have any self-references to initialize its values (no `this`), we can use this syntax instead (uses closure instead of a function block return):
```js
data: () => ({
  landingValid: false,
  operationTypes: [
    ...
  ]
}),
```